.wp-env.json

```diff
@@ -1,6 +1,66 @@
 {
 	"core": "WordPress/WordPress",
 	"plugins": [ "." ],
+	"excludes": [
+		"./node_modules",
+		"./packages/format-library/src",
+		"./packages/blocks/src",
+		"./packages/keycodes/src",
+		"./packages/block-serialization-default-parser/src",
+		"./packages/block-library/src",
+		"./packages/data-controls/src",
+		"./packages/jest-puppeteer-axe/src",
+		"./packages/interface/src",
+		"./packages/api-fetch/src",
+		"./packages/token-list/src",
+		"./packages/autop/src",
+		"./packages/viewport/src",
+		"./packages/server-side-render/src",
+		"./packages/react-native-aztec/src",
+		"./packages/babel-plugin-makepot/src",
+		"./packages/wordcount/src",
+		"./packages/plugins/src",
+		"./packages/shortcode/src",
+		"./packages/edit-navigation/src",
+		"./packages/e2e-test-utils/src",
+		"./packages/date/src",
+		"./packages/compose/src",
+		"./packages/edit-widgets/src",
+		"./packages/edit-site/src",
+		"./packages/element/src",
+		"./packages/nux/src",
+		"./packages/block-directory/src",
+		"./packages/url/src",
+		"./packages/priority-queue/src",
+		"./packages/notices/src",
+		"./packages/primitives/src",
+		"./packages/annotations/src",
+		"./packages/react-native-editor/src",
+		"./packages/warning/src",
+		"./packages/components/src",
+		"./packages/list-reusable-blocks/src",
+		"./packages/dom-ready/src",
+		"./packages/media-utils/src",
+		"./packages/html-entities/src",
+		"./packages/hooks/src",
+		"./packages/jest-console/src",
+		"./packages/icons/src",
+		"./packages/block-editor/src",
+		"./packages/dom/src",
+		"./packages/rich-text/src",
+		"./packages/docgen/src",
+		"./packages/a11y/src",
+		"./packages/core-data/src",
+		"./packages/edit-post/src",
+		"./packages/escape-html/src",
+		"./packages/i18n/src",
+		"./packages/data/src",
+		"./packages/blob/src",
+		"./packages/editor/src",
+		"./packages/keyboard-shortcuts/src",
+		"./packages/redux-routine/src",
+		"./packages/deprecated/src"
```

Comment:
Is this all packages except block-library? 
Feels ike we should tweak our setup to avoid loading php files from src folders and just ignore `/packages`

Reply:
> to avoid loading php files from src folders and just ignore /packages

I'm curious what benefit that gives us :)

Reply:
> Is this all packages except block-library?

It's all packages, as obtained via

```bash
find packages -type d -name src -d 2
```